changelog.md

```diff
@@ -1,7 +1,37 @@
-### 2.1.2
+### 2.2
```

Comment:
We are unfortunately well past GHC 9.12 fork and the agreement with GHC team was not to make major version bumps at this stage. My preference would be to stick to 2.1.2, even despite some substantial changes included. What do you think?

Reply:
Ah I didn't know that was decided. Fine by me.

Reply:
The rationale is that bumping a major version at this stage causes a chain reaction, because other boot packages, including those who have made a release already, would have to bump their upper bounds. Historically this has often caused delays in GHC release process.